cookbooks/aws-parallelcluster-slurm/templates/default/slurm/slurm.conf.erb

```diff
@@ -27,7 +27,7 @@ LaunchParameters=enable_nss_slurm
 # CLOUD CONFIGS OPTIONS
 <% if node['cluster']['use_private_hostname'] == 'true' or node['cluster']['dns_domain'].nil? or node['cluster']['dns_domain'].empty? -%>
 SlurmctldParameters=idle_on_node_suspend,power_save_min_interval=30,node_reg_mem_percent=<%= node['cluster']['slurm_node_reg_mem_percent'] %>
-TreeWidth=65533
+TreeWidth=30
```

Comment:
This is not where the default TreeWidth is set. This is a special case when the DNS resolution is disabled.